src/app/services/user-store.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class UserStoreService {
+  readonly userNameSubject = new BehaviorSubject<string>('Marcus Åström');
+  userName$ = this.userNameSubject.asObservable();
+
+  readonly placeOfWorkSubject = new BehaviorSubject<string>('IXI');
+  placeOfWork$ = this.placeOfWorkSubject.asObservable();
+
+  readonly notificationsSubject = new BehaviorSubject<
+    {
+      notification: string;
+      type: 'error' | 'success' | 'information' | 'warning';
+    }[]
+  >([
+    {
+      notification: 'You need to update your username.',
+      type: 'error',
+    },
+    {
+      notification: 'Your vacation request has been approved.',
+      type: 'success',
+    },
+    {
+      notification: 'You need to update your password.',
+      type: 'error',
+    },
+    {
+      notification: 'Your username is about to expire.',
+      type: 'warning',
+    },
+  ]);
```

Comment:
Should the notificationsSubject contain all types of notifications as default or only the latest?

Reply:
They should have defaults. Remember, this is just a demo/test page for us to validate that the component work in angular.